crates/wdk-build/rust-driver-sample-makefile.toml

```
@@ -0,0 +1,53 @@
+# This file is used to extend the standarrd rust-driver-makefile to build official sample drivers. See examples at https://github.com/microsoft/Windows-rust-drivers-samples
+# Using this file requires extending both the standard makefile and this makefile in order, as follows:
+# extend = [ { path =  "target/rust-driver-makefile.toml" }, { path = "target/rust-driver-sample-makefile.toml" } ]
+
+[tasks.wdk-samples-setup]
+private = true
+dependencies = ["wdk-build-init"]
+install_crate = { crate_name = "rust-script", min_version = "0.30.0" }
```

```suggestion
private = true
install_crate = { crate_name = "rust-script", min_version = "0.30.0" }
```

This dependency will cause init to run twice (once from dependency and once as the special init task